### DISCUSSION TITLE
CSV Contact

### DISCUSSION THREAD

**Discussion by andrucha97** [McShelby/hugo-theme-relearn#106] [Q&A] [answered]
I would like to share an Issue with you without doing it public. Maybe you can share an E-Mail or private Contact in the description?

**Comment by McShelby** [comment, marked as answer]
You can follow [this link to my web page](https://soeren-weber.de) and leave a comment. Note the form *does not* indicate that the email was successfully sent.

Alternativley you can sent me a message [via Twitter](https://twitter.com/DerAnglizist),

GitHub definatley is in need of direct messaging capabilities! ;-)

**Reply by andrucha97** [reply]
Hey. I wrote you a Mail. Hope you got it.

**Reply by McShelby** [reply]
Hm, not there yet (I doubt it will get thru). You can also try via Twitter DM @DerAnglizist

**Reply by andrucha97** [reply]
Ok. I just tried to send it you there :)

**Reply by McShelby** [reply]
Seems that my Twitter settings were to restrictive. Please try again. (Sadly GitHub doesn't offer a DM feature, yet)

**Reply by andrucha97** [reply]
Just sendet it again.

**Reply by McShelby** [reply]
Now it worked. I'll get back to you, soon via email.

**Reply by McShelby** [reply]
Hey @andrucha97, I've replied to your Twitter DM. Please get back to me regarding further information.
